src/coverlet.core/CoverageResult.cs

```diff
@@ -8,175 +8,181 @@
 
 namespace Coverlet.Core
 {
-    internal class BranchInfo
+  internal class BranchInfo
+  {
+    [System.Text.Json.Serialization.JsonInclude]
```

Comment:
I think we can drop the `JsonInclude` attribute if we add the `IncludeFields` property to the json serializer of the `JsonReporter`.
```csharp
// JsonReporter
      var options = new JsonSerializerOptions
      {
        IncludeFields = true,
        WriteIndented = true
      };
```